compiledtrees/compiled.py

```diff
@@ -28,33 +28,36 @@ class CompiledRegressionPredictor(object):
     http://courses.cs.washington.edu/courses/cse501/10au/compile-machlearn.pdf
     http://crsouza.blogspot.com/2012/01/decision-trees-in-c.html
     """
-    def __init__(self, clf):
-        self._n_features, self._evaluator, self._so_f = self._build(clf)
+    def __init__(self, clf, n_jobs=-1):
+        self.n_jobs = n_jobs
```

Comment:
Why do we need to track n_jobs as part of the state of the CompiledTree? It seems like this could be removed?